src/journal/test-audit-type.c

```diff
@@ -20,6 +20,7 @@ static void test_audit_type(void) {
                 print_audit_label(i);
 }
 
-int main(int argc, char **argv) {
+int main(void) {
```

Comment:
uh?

Reply:
Some test function are declared like this, this is allowed, and since the arguments are not used... I can revert this part if you want

Reply:
yes, please leave the normal function prototype in place here. I am aware that the prototype may differ, but it's always a bit ugly, since it just ignores the parameters that are passed anyway. That that works is kind of an ugly C feature if you ask me...